src/test/java/org/folio/sample/integration/api/books/GetLeapYearBookTest.java

```diff
@@ -0,0 +1,59 @@
+package org.folio.sample.integration.api.books;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
+
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.response.Response;
+import java.time.LocalDate;
+import java.util.List;
+import org.folio.sample.domain.dto.BookDTO;
+import org.folio.sample.domain.dto.BookForCreationDTO;
+import org.folio.sample.integration.AbstractBaseApiTest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class GetLeapYearBookTest extends AbstractBaseApiTest {
+  @Test
+  void testEmptyGet(){
```

Comment:
Make sure the style is consistent for functions, ie `testEmptyGet(){` vs `testEmptyGet() {`. I think convention is the second example, for this project.